src/components/WorkspaceAdd/WorkspaceAddButton.test.tsx

```diff
@@ -286,20 +297,28 @@ describe('Add Workspace modal', () => {
         },
       },
     });
-    await waitFor(() => expect(screen.queryByRole('dialog')).not.toBeInTheDocument());
+
+    await waitFor(() => expect(mockedUsedNavigate).toHaveBeenCalledWith('1workspace'));
+    expect(screen.queryByRole('dialog')).not.toBeInTheDocument();
   });
 
-  it('Clicking cancel button closes modal', () => {
+  // test('Modal shows error if addition is successfully created but unable to verify workspace name', () => {
```

Comment:
@kdoberst why did we disable this test?

Reply:
I messed up.  I forgot to implement the actual test.  😕   I'll add that in the next commit